downstream/modules/platform/proc-controller-user-permissions.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc-controller-user-permissions"]
+
+= Adding and removing user permissions
+
+To add permissions to a particular user:
+
+.Procedure
+. From the *Users > Details* page, click btn:[Add], which opens the *Add Permissions* wizard.
```

Comment:
Suggested re-write:

. From the *Users* list view, click on the desired user.
. On the *Details* tab click btn:[Add]. This opens the *Add Permissions* wizard.